python/addons/embed_bert.py

```diff
@@ -1249,6 +1249,19 @@ def __init__(self, **kwargs):
         self.wordpiece_tok = WordpieceTokenizer(load_vocab(kwargs.get('vocab_file')))
         self.mxlen = kwargs.get('mxlen', -1)
 
+    def count(self, tokens):
```

Comment:
Does this mean that the `count` method should be moved into the `AbstractVectorizer` instead of being in the `Token1DVectorizer`?